### PR TITLE
ENH: validate the argument `width` on `Styler.bar`

### PR DESCRIPTION
**Pull request by attack68**

### CODE REVIEW

File: pandas/io/formats/style.py [hunk 1]
```diff
@@ -2121,6 +2121,9 @@ def bar(
                 "(eg: color=['#d65f5f', '#5fba7d'])"
             )
 
+        if not (0 <= width <= 100):
```
Comment (jreback) [inline pandas/io/formats/style.py:2124]: height? other ones?

Comment (attack68) [reply pandas/io/formats/style.py:2124]: will add height after other PR which adds feature

**Comment by jreback** [comment]
can you add the height validation here (just merged that PR)

**Comment by attack68** [comment]
done and green

**Comment by jreback** [comment]
thanks!

**Pull request merged**